packages/docusaurus-plugin-content-docs/src/metadata.ts

```diff
@@ -104,7 +104,14 @@ export default async function processMetadata({
   }
 
   if (showLastUpdateAuthor || showLastUpdateTime) {
-    const fileLastUpdateData = lastUpdate(filePath);
+    // Use fake data in dev for faster development
+    const fileLastUpdateData =
```

Comment:
Should this be done within `lastUpdate`?

Reply:
If its put in lastupdate, it wont be pure function anymore and it will always return fake data